app.js

```diff
@@ -71,7 +71,7 @@ cardData.forEach((card, index) => {
   const imageContainer = document.createElement('div');
```

Comment:
Your images are broken it would be nice to add links that are not broken. 

This is your design:

![Screenshot 2023-04-29 at 1 05 46 PM](https://user-images.githubusercontent.com/79506105/235300493-8ad4b835-e832-40d2-8838-23a1819f5a9f.png)

Please stick to this design:

![Uploading Screenshot 2023-04-29 at 5.53.48 AM.png…]()